job-dsls/jobs/kie_release_jobs.groovy

```diff
@@ -458,7 +389,7 @@ matrixJob("serverMatrix-kieReleases-${kieVersion}") {
             properties("maven.test.failure.ignore": true)
             properties("deployment.timeout.millis":"240000")
             properties("container.startstop.timeout.millis":"240000")
-            properties("eap64x.download.url":"http://download.devel.redhat.com/released/JBEAP-6/6.4.4/jboss-eap-6.4.4-full-build.zip")
+            properties("eap7.download.url":"http://download.eng.brq.redhat.com/released/JBEAP-7/7.0.2/jboss-eap-7.0.2-full-build.zip")
```

Comment:
Let's use the generic download.devel. The specific server will be picked up based on the location. Downloading stuff from BRQ into RDU will be slower.